src/open_inwoner/accounts/views/invite.py

```diff
@@ -40,4 +42,11 @@ def get_object(self, queryset=None):
             self.log_system_action(_("invitation used"), invite)
             raise Http404(_("The invitation was already used"))
 
+        if user.is_authenticated:
+            if user.email != invite.invitee_email:
```

Comment:
Comments:
- I would suggest to remove the check on the emailadres, this to avoid the situation where the user registers under a different emailadres than that the invite was sent. You can also do a case-insensitive comparison if you think this is too lax.
- After accepting and saving the invite, you can add request.user to the contacts of the user sending the invite. You can then return the accepting user to Mijn Contacten, with a message that the user has accepted the invitation.